bouquet/config.py

```
@@ -5,11 +5,12 @@
 from typing import Optional
 
-# Import MethodType and SUPPORTED_METHODS from calculator (single source of truth)
-from bouquet.calculator import MethodType, SUPPORTED_METHODS
+# Method names are registry-driven in calculator.py; MethodType is just the str
+# alias used for the energy/optimizer config field hints. The selectable set is
+# CalculatorFactory.available_methods() (the installed subset).
+from bouquet.calculator import MethodType
 
-# Re-export for backwards compatibility
-__all__ = ["MethodType", "SUPPORTED_METHODS", "Configuration"]
+__all__ = ["MethodType", "Configuration"]
```

Ruff (0.15.18)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)